keel-igor/src/main/kotlin/com/netflix/spinnaker/igor/ArtifactService.kt

```diff
@@ -6,9 +6,14 @@ import retrofit2.http.Path
 
 interface ArtifactService {
 
-  @GET("/artifacts/rocket/{application}/{version}")
+  @GET("/artifacts/rocket/{packageName}/{version}")
```

Comment:
this is actually the package name and not the application. renamed the parameter so that was clear.